projects/pgai/tests/vectorizer/conftest.py

```diff
@@ -46,7 +46,7 @@ def vcr_():
     )
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="class")
```

Comment:
Took me a bit to understand this change, but I suppose the existing CLI tests contained a DB reset which is why we don't need a new container for each one.

I'm not so sure about the class grouping. I feel like we should either make the fixture sessions scoped and reset the DB every time. Or always create a new container.
Now it's in a bit of a weird mixed state, but well it works.